tests/pyccel/test_pyccel.py

```diff
@@ -229,10 +253,13 @@ def pyccel_test(test_file, dependencies = None, compile_with_pyccel = True,
             dependencies = []
         if language=='fortran':
             compile_fortran(cwd, test_file, dependencies)
-        else:
+        elif language == 'c':
             compile_c(cwd, test_file, dependencies)
 
-    lang_out = get_lang_output(get_exe(test_file))
+    if language == "python":
+        lang_out = get_python_output(get_exe(test_file, language, prog=prog))
```

Comment:
I'm not sure this line makes sense if `prog=False`? If it is not a program then nothing will execute so there will be no output to collect, or am I missing something?

Reply:
i set the value of prog when its a program like in the line 639 which is a program

Reply:
This explains why you need `prog=True`, but not why you need `prog=False` if `prog` is always true (this seems to be the assumption made for fortran/c) then this argument adds nothing

Reply:
i see, no need for the prog argument, I removed it.